pkg/skaffold/kubernetes/portforward/kubectl_forwarder.go

```diff
@@ -203,7 +209,10 @@ func (*KubectlForwarder) monitorErrorLogs(ctx context.Context, logs io.Reader, c
 				if err := cmd.Terminate(); err != nil {
 					logrus.Tracef("failed to kill port forwarding %v, err: %s", p, err)
 				}
+				err <- fmt.Errorf("port forwarding %v got terminated: output: %s", p, s)
 				return
+			} else if strings.Contains(s, "Forwarding from") {
```

Comment:
shd we check if `err == nil` 

Is the expectation to write  `nil` to channel when
1. Port forwarding was successful and return
2. Port forwarding ran into error due to port being busy but keep re-trying?